internal/config.go

```diff
@@ -59,13 +60,17 @@ func DefaultConfig() *Config {
 	}
 
 	return &Config{
-		SourceDir: ".",     // Current directory by default
-		TargetDir: homeDir, // User's home directory by default
-		BackupDir: filepath.Join(homeDir, ".dloom/backups"),
-		Force:     false,
-		Verbose:   false,
-		DryRun:    false,
-		Packages:  make(map[string]*PackageConfig),
+		SourceDir:      ".",     // Current directory by default
+		TargetDir:      homeDir, // User's home directory by default
+		BackupDir:      filepath.Join(homeDir, ".dloom/backups"),
+		Force:          false,
+		Verbose:        false,
+		DryRun:         false,
+		Packages:       make(map[string]*PackageConfig),
+		IgnorePackages: []string{
```

Comment:
Today we do not have a link --all command, that means for this to take effect, users has to explicitly run `link .git` but then we have hardcoded to ignore it.

Reply:
No, this is when you run:
1. On '.' (Root) of the repository to plain sync all dotfiles you have in the repo. Which contains a .git folder.
2. Run it on some directory which inside it could have a .git folder.

See the tests

Reply:
Hmmm, ok. Then also add some more ignores, like vscode stuff?

Reply:
Unaware of what . files vscode creates. Will keep adding them.
Another one is .dsstore